frontend/src/metabase/parameters/components/Parameters/Parameters.jsx

```diff
@@ -25,17 +25,12 @@ export default class Parameters extends Component {
 
     if (this.props.syncQueryString) {
       // sync parameters to URL query string
-      const queryParams = {};
-      for (const parameter of collateParametersWithValues(
+      const parameterValuesBySlug = getParameterValuesBySlug(
         parameters,
         parameterValues,
-      )) {
-        if (parameter.value) {
```

Comment:
I think this code might've been slightly wrong, since I _think_ `parameter.value` can be `0` for a number filter.